metafora.go

```diff
@@ -71,7 +71,7 @@ func NewConsumer(coord Coordinator, h HandlerFunc, b Balancer) (*Consumer, error
 //
 // Run blocks until Shutdown is called or an internal error occurs.
 func (c *Consumer) Run() {
-	Debug("Starting consumer")
+	Debug(c, " Starting consumer")
```

Comment:
This is the hacky way I used coordinator's String() method as a weak "contextual" logger. Consumer's String() just calls the coordinator's.... it's very useful in output where there's more than 1 consumer in play, but I'm not sure that justifies an ugly implementation.